### PR TITLE
[Feature] Add functionality to collect and aggregate top news headlines across different sectors. 🌐

### PR DESCRIPTION
**Pull request by baiden00**
### Background
This addition uses the python News API to aggregate top news headlines across different sectors. AutoGPT users can get top news headlines instantly with one command

### Changes
 I created a command to aggregate the news in a new file called news_agggregator.py. I used the decorator and registered the command with the registry. I added instructions in the README file and the API library to the requirements.txt. I used concurrency to speed up the requests just in case

### Documentation

Changes are documented in the README by informing users of the need to obtain a free tier API key required for the news aggregation to work.

### Test Plan
I tested the new aggregator on my own by obtaining the API KEY and ran my code separately before integrating it to AutoGPT. No merge conflicts as at this time.

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: autogpt/commands/news_aggregation.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from autogpt.commands.command import command
+from autogpt.config import Config
+
+CFG = Config()
+
+categories = ["technology", "business", "entertainment", "health", "sports", "science"]
+news_api_client = NewsApiClient(CFG.news_api_key)
+
+
+def get_news_headlines_for_category(category: str):
```
Comment (zhuyuecai) [inline autogpt/commands/news_aggregation.py:14]: type hint for return type?

Comment (baiden00) [reply autogpt/commands/news_aggregation.py:14]: done

Comment (zhuyuecai) [reply autogpt/commands/news_aggregation.py:14]: I mean, it should be `def get_news_headlines_for_category(category: str) -> list[str]:`

**Comment by ntindle** [comment]
Great work! I'd love to see this moved into the plugins repo over here: https://github.com/Significant-Gravitas/Auto-GPT-Plugins

We're working on getting documentation up for it, but for now, just follow the pattern of the twitter plugin!

**Review by ntindle — CHANGES_REQUESTED** [review]
Please move to the plugins repo https://github.com/Significant-Gravitas/Auto-GPT-Plugins

**Comment by ntindle** [comment]
Closing in favor of it being a first party plugin in the plugins repo

**Pull request closed without merging**